miniwallet/test/Mini721.ts

```diff
@@ -0,0 +1,971 @@
+import { expect } from 'chai'
+import { ethers, network } from 'hardhat'
+import { ADDRESS_ZERO, prepare721, userMint721, communityMint721 } from './utilities'
+import { BigNumber } from 'ethers'
+
+// let snapshotId: string;
+
+describe('Mini721', function () {
+  before(async function (this) {
+    await prepare721(this, [])
+  })
+
+  beforeEach(async function (this) {
+    this.snapshotId = await ethers.provider.send('evm_snapshot', [])
+    // console.log(`New Snapshot beforeEach: ${snapshotId}`);
+    this.mini721 = await this.Mini721.deploy(
+      false, // saleIsActive
+      false, // metadataFrozen
+      false, // provenanceFrozen
+      10000, // maxMiniTokens
+      ethers.utils.parseEther('0.1'), // mintPrice 0.1 ETH
+      10, // maxPerMint
+      '', // "ipfs://QmPcY4yVQu4J2z3ztHWziWkoUEugpzdfftbGH8xD49DvRx/", // baseUri
+      '' // "ipfs://Qmf8WkAVZtkBwngG4mTrPk23vDd6z8dZW2UshV9ywWGyB9/contract.json" // _contractUri
+    )
+    await this.mini721.deployed()
+    // console.log("Mini721 contract", this.mini721.address);
+  })
+
+  afterEach(async function (this) {
+    // console.log(`Reverting Snapshot : ${snapshotId}`);
+    await network.provider.send('evm_revert', [this.snapshotId])
+  })
+  // TODO
+  // NFT Build Process (Images, Metadata, Provenance Hash)
+
+  // Deployment Tests
+  describe('validate Initial Deploy', function (this) {
+    it('Mini721 contract should be created', async function () {
+      expect(await this.mini721.address).to.equal(
```

Comment:
this might be too specific

Reply:
Changed to `.to.be.properAddress;` 
https://ethereum-waffle.readthedocs.io/en/latest/matchers.html#proper-address
🙏